cli/src/commands/prev.rs

```diff
@@ -105,13 +96,11 @@ pub(crate) fn cmd_prev(
     let target = match targets.as_slice() {
         [target] => target,
         [] => {
-            return Err(user_error(format!(
-                "No ancestor found {offset} commit{} back",
-                if offset > 1 { "s" } else { "" }
-            )))
+            return Err(user_error("No ancestor found at requested offset."));
```

Comment:
The previous message was definitely clearer.

Reply:
And worse for i18n, which came up on another PR. IMO the user doesn't need to be reminded of the offset the specified because they just typed it into the CLI and it is probably still visible.

But I don't really care about the message. We can use the old one if people prefer.